app/mailers/planning_application_mailer.rb

```diff
@@ -3,6 +3,17 @@
 class PlanningApplicationMailer < Mail::Notify::Mailer
   NOTIFY_TEMPLATE_ID = "7cb31359-e913-4590-a458-3d0cefd0d283"
 
+  def email_applicant_and_agent(planning_application)
```

Comment:
applicant_and_agent_email, otherwise it sounds like it's going to actually mail them